docs/diagrams/FindSequenceDiagram.puml

```diff
@@ -0,0 +1,74 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":FindCommandParser" as FindCommandParser LOGIC_COLOR
+participant ":ParserUtil" as ParserUtil LOGIC_COLOR
+participant ":NameContainsKeywordPredicate" as NameContainsKeywordPredicate LOGIC_COLOR
```

Comment:
might be good to have the predicate be an instantiated object? something like predicate:NameContainsKeywordPredicate instead?